src/components/Dashboard/CompleteFiling.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="complete-ia-filing body-2">
+    <h4>Incorporation Complete</h4>
+
+    <p>{{name}} has been successfully incorporated.</p>
+    <p>Return to your Manage Businesses dashboard to access your business and file changes.</p>
```

Comment:
Mismatch.
The text says Manage Businesses dashboard but the URL is the Business Registry home page.

If you really meant the Manage Businesses dashboard then look at https://github.com/bcgov/business-filings-ui/blob/01b3c1d1eeecc23e5b40d9fc82478d68dd8aea7b/src/components/Dashboard/TodoList.vue#L881 .

Reply:
ahh good catch, my brain glossed over the 'business' part